Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -110,17 +134,33 @@ public class RocksDBStore : BaseStore
                 _options = _options.SetMaxLogFileSize(maxLogFileSizeValue);
             }
 
-            _blockDb = RocksDBUtils.OpenRocksDb(_options, RocksDbPath(BlockDbName));
+            _blockIndexDb = RocksDBUtils.OpenRocksDb(_options, BlockDbPath(BlockIndexDbName));
             _blockPerceptionDb =
                 RocksDBUtils.OpenRocksDb(_options, RocksDbPath(BlockPerceptionDbName));
-            _txDb = RocksDBUtils.OpenRocksDb(_options, RocksDbPath(TxDbName));
+            _txIndexDb = RocksDBUtils.OpenRocksDb(_options, TxDbPath(TxIndexDbName));
             _stagedTxDb = RocksDBUtils.OpenRocksDb(_options, RocksDbPath(StagedTxDbName));
 
             // When opening a DB in a read-write mode, you need to specify all Column Families that
             // currently exist in a DB. https://github.com/facebook/rocksdb/wiki/Column-Families
             var chainDbColumnFamilies = GetColumnFamilies(_options, ChainDbName);
             _chainDb = RocksDBUtils.OpenRocksDb(
                 _options, RocksDbPath(ChainDbName), chainDbColumnFamilies);
+
+            _rwTxLock = new ReaderWriterLockSlim(LockRecursionPolicy.SupportsRecursion);
+            _rwBlockLock = new ReaderWriterLockSlim(LockRecursionPolicy.SupportsRecursion);
+
+            _blockDbCache = new LruCache<string, RocksDb>();
```

Comment:
is it okay we use default `LruCache<T>`'s?

Reply:
`LruCache<T>`'s default cache size is 1000, you think it is too much?

Reply:
I'm parameterized `LruCache<T>`'s capacity.